services/zip/internal/cmd/meta.go

```diff
@@ -0,0 +1,50 @@
+package main
+
+import (
+	def "go.beyondstorage.io/v5/definitions"
+	"go.beyondstorage.io/v5/types"
+)
+
+var Metadata = def.Metadata{
+	Name:  "zip",
+	Pairs: []def.Pair{},
+	Infos: []def.Info{},
+	Factory: []def.Pair{
+		def.PairWorkDir,
+	},
+	Service: def.Service{},
+	Storage: def.Storage{
+		Features: types.StorageFeatures{
+			WriteEmptyObject: true,
+
+			Create:   true,
+			Delete:   true,
+			List:     true,
+			Metadata: true,
+			Read:     true,
+			Write:    true,
```

Comment:
Maybe we should keep the default values of the features before implementing them?